IntegrationTests/ContentBlockerTests.swift

```diff
@@ -119,9 +120,13 @@ class ContentBlockerTests: XCTestCase {
         let navBar = app.navigationBars["DuckDuckGo.MainView"]
         navBar.otherElements["siteRating"].tap()
     }
+ 
+    private func waitForLaunch() {
+        sleep(10)
+    }
     
-    private func waitForPageTitle() {
-        sleep(2)
+    private func waitForPageLoad() {
+        sleep(6)
```

Comment:
Optional improvement: Have a look in SnapshotHelper.swift for a way to watch the loading indicator.  You'll probably have to add your own timeout though.

Reply:
I like that, thanks!